index.html

```diff
@@ -0,0 +1,287 @@
+              bulletproof code samples. Written by Heydon Pickering.
+            </p>
+            <a href="#" class="inclusive-components-child"
+              >Learn more about the book</a
+            >
+            <img
+              class="inclusive-components-child"
+              src="https://d33wubrfki0l68.cloudfront.net/c1ce65098d4e30daafb0e957ba930e43347f72aa/975a1/images/books/inclusive-design-components.svg"
+              alt="Smashing-puppet"
+            />
+          </div>
+        </div>
+        <div></div>
+        <div></div>
+      </section>
+    </main>
+    <div class="form1">
+      <div class="wide form1-content darkgrey-black">
+        <h2>Person Of The Week</h2>
+      </div>
+      <article class="thumbnail grey-black">
+        <h2>Gradient Color Hints</h2>
+      </article>
+      <article class="thumbnail ligth-grey">
+        <h2>CSS Architecture For JavaScript Applications</h2>
+      </article>
+      <article class="thumbnail grey-black">
+        <h2>Design Conferences 2020</h2>
+      </article>
+      <article class="thumbnail ligth-grey">
+        <h2>Dynamic, Data-Based Color</h2>
+      </article>
+      <article class="thumbnail grey-black">
+        <h2>Arrow Functions Cheatsheet</h2>
+      </article>
+      <article class="thumbnail ligth-grey">
+        <h2>Gradient Color Hints</h2>
+      </article>
+      <div class="wide form1-more"><a href="#">More articles</a></div>
+    </div>
+    <div class=" full-width form2 light-grey design-items-parent">
+      <div class="design-items">
+        <h2>Design Systems</h2>
+        <p>
+          Meet Design Systems, a recent Smashing book in which Alla Kholmatova
+          explores how to set up an effective design system to create great
+          digital products. With common traps, gotchas and lessons learned.
+        </p>
+      </div>
+      <div class="design-items">
+        <img
+          src="https://res.cloudinary.com/indysigner/image/upload/v1542959380/shop/design-systems-hardcover-book.png"
+          alt="Book"
+        />
+      </div>
+    </div>
+    <div class="form1-container">
+      <article class="thumbnail light-grey">
+        <h2>New Front-End Challenges To Master</h2>
+      </article>
+      <article class="thumbnail light-grey">
+        <h2>Performance Optimization & Strategy</h2>
+      </article>
+      <article class="thumbnail light-grey">
+        <h2>Improving Workflow & Data Protection</h2>
+      </article>
+
+      <div class="aside">
+        <div class="thumbnail-aside darkgrey-black">
+          <h2>Design Patterns & Checklists</h2>
+        </div>
+      </div>
+
+      <div class="large white-black">
+        <h2>Smashing Newsletter</h2>
+        <img
+          src="https://d33wubrfki0l68.cloudfront.net/ce7741e1659c0366fda51555365194e0bfd136c0/ae578/images/smashing-cat/cat-with-trampet.svg"
+          alt="image"
+        />
+      </div>
+    </div>
```

Comment:
The horizontal scroll is coming from the styling/sizing for this div (`.form1-container`). Re-check  your styling for this div to fix the horizontal scroll.